### PR TITLE
Features

### PR DESCRIPTION
**Pull request by quavo19**
### in this section my partener @outkarm, and I @Donald ;

- **Followed all CSS HTML and javascript best practices to make navigation for our Awesome books web app;**
- **We modified the Awesome books application to have:**

> **A Navigation bar.**  

> **Three content sections**

`Books list.` 🔢 
`Add book form.` 📖  and
`Contact info.` ☎️ 

- We made sure that,  when you click on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,19 +5,70 @@
...
+      href="https://fonts.googleapis.com/css2?family=Nerko+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Heebo:wght@600&display=swap"
+      rel="stylesheet"
+    />
     <title>Awsome Books</title>
```
Comment (OyePriscilla) [inline index.html:20]: - [optional] You may want to change the title to a more reader text. 😄

File: index.html [hunk 2]
```diff
@@ -5,19 +5,70 @@
...
+        <h1>Add a new book</h1>
+        <form>
+          <input type="text" id="Title" placeholder="Title" /><br /><br />
+          <input type="text" id="Author" placeholder="Author" /><br /><br />
+          <input type="button" id="Add-Button" value="Add" />
+        </form>
+      </div>
+      <div class="contact">
+        <div class="info">
+          <h1>Contact information</h1>
```
Comment (OyePriscilla) [inline index.html:55]: - [optional] For SEO optimization, it is always better to have one `h1` tag in an HTML document. Kindly change this to other header tag aside `h1`. Thanks. ℹ️

**Review by OyePriscilla — APPROVED** [review]
## Hi @quavo19 and @Outkarm   :smiley: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Descriptive pull request message and title :heavy_check_mark: 
Linter error checks validated :heavy_check_mark: 
Commit messages are meaningful. :heavy_check_mark: 
GitHub flow observed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (quavo19) [reply index.html:55]: I WILL LOOK INTO THAT

**Pull request merged**